core/src/main/java/com/rudderstack/android/sdk/core/Constants.java

```diff
@@ -32,6 +32,12 @@ class Constants {
     static final boolean TRACK_LIFECYCLE_EVENTS = true;
     // whether we should record screen views automatically
     static final boolean RECORD_SCREEN_VIEWS = false;
+    // minimum duration for inactivity is 0 milliseconds
+    static final int MIN_SESSION_TIMEOUT = 0;
+    // default duration for inactivity is 5 minutes or 300000 milliseconds
+    static final int DEFAULT_SESSION_TIMEOUT = 300000;
```

Comment:
milliseconds data type should be long

Reply:
This is static. I think we can use `int` here.